src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -63,8 +65,20 @@
   public static final String TABLE_NAME = "dataset";
 
   private static final String summaryQueryColumns =
-      " dataset.id, name, description, default_profile_id, project_resource_id, "
-          + "application_resource_id, secure_monitoring, created_date, ";
+      " dataset.id, dataset.name, description, default_profile_id, project_resource_id, "
+          + "dataset.application_resource_id, secure_monitoring, created_date, ";
+
+  private static final String summaryCloudPlatformColumns =
+      " project_resource.id as google_project_id, "
+          + "storage_account_resource.name as storage_account_resource_name, ";
+
+  private static final String summaryCloudPlatformJoins =
+      "LEFT JOIN project_resource "
```

Comment:
Careful: this will break if you have multiple projects/storage accounts per dataset (which can happen if you ingest using a different billing profile.  You'll probably need to do a JSON aggregation)

Reply:
Good catch.